external/payara-mp-tck/README.md

```diff
@@ -0,0 +1,36 @@
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+# External Payara-mp-tck Tests
+Payara Platform is cloud-native, optimized for production environments.
```

Comment:
Can remove this line.

Reply:
@ShelleyLambert thank you.
I have removed the content_key.
Tomorrow I will try running the test myself with those steps because I want to learn.
Thank you.
Do have a wonderful night rest.